src/main/java/edu/carroll/cs389application/service/UserServiceImpl.java

```diff
@@ -0,0 +1,81 @@
+package edu.carroll.cs389application.service;
+
+import edu.carroll.cs389application.jpa.model.Login;
+import edu.carroll.cs389application.jpa.repo.LoginRepo;
+import edu.carroll.cs389application.web.form.LoginForm;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+/**
+ * Implementation of UserService interface for user login and registration.
+ */
+@Service
+public class UserServiceImpl implements UserService {
+    /**
+     * This is a basic logging library that we use to keep track of logs for application
+     */
+    private static final Logger log = LoggerFactory.getLogger(UserServiceImpl.class);
+
+    /**
+     * Repository interface for user entities. Provides CRUD functionality and
+     * additional methods for finding Login entities by username.
+     */
+    private final LoginRepo loginR;
```

Comment:
Nit: Try to use the same naming convention as other classes.  

`imageRepo` `loginR`.